RELEASE_NOTES.md

```diff
@@ -34,4 +34,23 @@ not designed to be a general solution to performance issues inside of chef-clien
 
 This debugging feature will mostly be useful to people who are already Ruby experts.
 
+### New ksh resource
 
+ksh is now available to use as a resource and is available as an interpreter in the script resource.
+
+Please see the DOC_CHANGES for more details.
```

Comment:
The DOC_CHANGES.md doesn't appear to have more details and just points you here: https://docs.chef.io/release/12-6/resource_ksh.html

might as well just link there instead.